api/novelist.py

```diff
@@ -498,13 +504,18 @@ def create_item_object(self, object):
         elif object[2] is not None:
             isbn = object[2]
 
+        title = object[3]
         mediaType = Edition.medium_to_additional_type[object[4]]
+        author = object[5].split(';')[0] if object[5] else object[6]
```

Comment:
Then, instead of this, pull an author name from object[8] `if role in Edition.AUTHOR_ROLES` and add it to a list. Use that list as the value of `Author`.

Reply:
If we have a Contributor but we're missing the sort name then I would just ignore it -- it won't do NoveList any good.

Reply:
Novelist's documentation says the Author is a string value, but it's a good question to ask to see if they can accept a list, too.